### PR TITLE
feat: add support for article section

### PR DESCRIPTION
**Pull request by killanaca**

### CODE REVIEW

File: packages/article/article-body/article-body.web.js [hunk 1]
```diff
@@ -3,11 +3,19 @@ import PropTypes from "prop-types";
...
-
+  const { section, content: bodyContent } = props;
+  const contentArray = bodyContent.map((data, index) => {
+    const item = {
+      data,
+      index
+    };
+    if (data.name === "ad") {
+      item.data.attributes = Object.assign({}, item.data.attributes, {
+        section
```
Comment (fourlastor) [inline packages/article/article-body/article-body.web.js]: should we add the section if it's not present? (it's not a required prop)

or should it be required?

File: packages/article/data-helper.js [hunk 2]
```diff
@@ -17,11 +17,19 @@ const prepareDataForListView = articleData => {
...
   ].concat(
-    articleData.content.map((i, index) => ({
-      type: "articleBodyRow",
-      data: i,
-      index
-    }))
+    articleData.content.map((i, index) => {
+      const item = {
+        type: "articleBodyRow",
+        data: i,
```
Comment (fourlastor) [inline packages/article/data-helper.js]: maybe we can rename `i` to `data` (or to whatever `i` is currently)

File: packages/article/article-body/article-body.web.js (hunk 1 above)
Comment (killanaca) [reply packages/article/article-body/article-body.web.js]: if it not present, we're gonna use the default value from the ads component

File: packages/article/article-body/article-body.web.js [hunk 3]
```diff
@@ -3,11 +3,19 @@ import PropTypes from "prop-types";
...
-  }));
-
+  const { section, content: bodyContent } = props;
+  const contentArray = bodyContent.map((data, index) => {
+    const item = {
+      data,
+      index
+    };
+    if (data.name === "ad") {
+      item.data.attributes = Object.assign({}, item.data.attributes, {
```
Comment (craigbilner) [inline packages/article/article-body/article-body.web.js]: while it's not an eslint rule, perhaps object spread would be more consistent?

**Pull request merged**